appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -128,6 +130,10 @@ export class CaseDetailsPage extends Page {
 		this.elements.addRelatedAppeals().click();
 	}
 
+	clickManageLinkedAppeals() {
```

Comment:
As this is using an element that exists in `BasePage` which `CaseDetailPage` inherits from I'd say it's worth considering moving this method (`clickManageLinkedAppeals`) to be within `BasePage` as it would allow that method to be reused effectively.

If you did this then you'd still be able to address it as you do now from within `unlinkAppeals.spec.js` so that code wouldn't change

Reply:
we agreed that as there is no existing element in base page we don't need to do this